core/authz.go

```diff
@@ -61,4 +62,25 @@ var policyByRoute = map[string][]string{
 
 	"/raft/join": {"internal"},
 	"/raft/msg":  {"internal"},
+
+	"/":                                       {"public"},
+	"/dashboard":                              {"public"},
+	"/dashboard/core":                         {"public"},
```

Comment:
I had this same thought, but I'm not sure how we'd do it safely--since we want the default to be unauthorized, but making "/" public would make the default public. (I guess we could explicitly use regular expressions?)